src/pages/Admin/Admin.tsx

```diff
@@ -81,6 +81,7 @@ const VenueList: React.FC<VenueListProps> = ({
   selectedVenueId,
   roomIndex,
 }) => {
+  // @debt This selector relies on all venues in firebase being loaded into memory.. not very efficient
   const venues = useSelector(orderedVenuesSelector);
```

Comment:
[note, ?OOS?] `VenueList` is rendered by `Admin` which is used in:

- `src/components/organisms/AppRouter/AppRouter.jsx`

Reply:
OOS, will handle this in a followup PR.